proto/forch_configuration.proto

```diff
@@ -123,6 +126,18 @@ message OrchestrationConfig {
     // Port to listen on for RADIUS responses
     int32 source_port = 4;
   }
+
+  // encapsulating orchestrated testing configurations
+  message OrchestratedTestingConfig {
+    // testing segment. If defined, automatic sequestering is enabled
+    string testing_segment = 11;
+
+    // range of testing VLANs, represented in a list of starting and ending VLAN
```

Comment:
is it a range or a list?  If it's a range then there should be two specific values. If it's an arbitrary list then it would be repeated.

Reply:
Separated it into two values.